Zend/tests/attributes/deprecated_001.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+#[Deprecated] attribute
+--FILE--
+<?php
+
+error_reporting(E_ALL | E_DEPRECATED);
```

Comment:
nit: As of PHP 8.0, E_ALL includes E_DEPRECATED, so this is redundant in a test targeting 8.0